pkgs/applications/misc/bukut/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, python3, fetchFromGitHub }:
+
+with python3.pkgs; buildPythonApplication rec {
+  version = "0.11";
+  pname = "bukut";
+
+  src = fetchFromGitHub {
+    owner = "peterjschroeder";
+    repo = "bukut";
+    rev = "9d735490dae9f657a74f33f2eb68656b31a5c5d8";
```

Comment:
Why are we not using the tag?

Reply:
It's a small github project without any tags

Reply:
```suggestion
    # project doesn't have tags
    rev = "9d735490dae9f657a74f33f2eb68656b31a5c5d8";
```

Reply:
Fixed